src/messages.rs

```diff
@@ -22,6 +32,64 @@ pub trait MessageTrait {
 pub struct Message {
     pub header: Header,
     pub payload: Payload,
+
+    pub metadata: TransportData,
+}
+
+/// Defines a transport-related properties that determines how the message
+/// will be broadcast.
+/// TODO: This should be moved out of consensus message definition.
+#[derive(Debug, Clone)]
+pub struct TransportData {
+    pub height: u8,
+    pub src_addr: String,
+}
+
+impl Default for TransportData {
```

Comment:
This default implementation should not exists

Reply:
Done.